azurerm/resource_arm_container_group.go

```diff
@@ -265,12 +259,33 @@ func flattenContainerGroupContainers(containers *[]containerinstance.Container)
 		}
 		// protocol isn't returned in container config
 
+		if len(*container.EnvironmentVariables) > 0 {
```

Comment:
this'll crash if `container.EnvironmentVariables` is nil - could we add an if statement around this?

Reply:
It's never nil, but a check is warranted to be safe. Done.

Reply:
To add some context to why this check is needed: if the Swagger is incorrect (which historically has happened a lot) this would be mapped incorrectly and crash